ucsb-cs156-f24/team02-f24-02
# Leozhu0 Menu Items Index Page

Create Page for menu items is complete. Please merge https://github.com/ucsb-cs156-f24/team02-f24-02/pull/97 first. The link to view the storybook is as follows: https://6726c65fac869bba81d3d68d-uhpzhtdsmq.chromatic.com/?path=/docs/pages-ucsbdiningcommonsmenuitems-ucsbdiningcommonsmenuitemsindexpage--docs

To test:
1. Run mvn spring-boot:run and npm start
2. Navigate to http://localhost:8080/ucsbdiningcommonsmenuitems
3. After adding an item, the table should populate with that item.
4. Sign in with an admin account to view Edit and Delete buttons

<img width="1396" alt="Screenshot 2024-11-10 at 9 52 08 PM" src="https://github.com/user-attachments/assets/25882e51-1629-43d4-89a1-d9bef36ac0b9">
